webhook_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func Test_webhook(t *testing.T) {
+	wh := &GithubWebhookHandler{
+		secret: "a1b2c3d4e5",
+	}
+
+	body := []byte(`{"foo":"bar", "action": "foo"}`)
+	signature := wh.computeHMAC(body, wh.secret)
+
+	t.Run("validate signature", func(t *testing.T) {
+
+		if !wh.isValidSignature(body, signature) {
+			t.Errorf("isValidSignature() expected true")
+		}
```

Comment:
I find it more readable and concise when using aserts/requires
```suggestion
		assert.True(t, wh.isValidSignature(body, signature))
```

Reply:
the reason for not using assert is same as `urfave/cli` its an 3rd party package. I normally use `google/go-cmp` for big struct diffs in test because it gives nice output. 
but in this case its simple equality check hence I didnt.